msbuild/Xamarin.MacDev.Tasks.Core/Tasks/CodesignTaskBase.cs

```diff
@@ -74,6 +76,9 @@ protected override string GenerateCommandLineCommands ()
 				args.AddQuoted (Path.GetFullPath (Entitlements));
 			}
 
+			if (DisableTimestamp)
+				args.Add ("--timestamp=none");
```

Comment:
TBH that surprised me a bit (from Vincent's document) as a timestamp is _generally_ something you had to a signature, yet it seems that removing it also removes the signing requirement. I'm glad but still surprised :)